### DISCUSSION TITLE
Support conversion of webp

### DISCUSSION THREAD

**Discussion by RunsOnRum** [visioncortex/vtracer#65] [Q&A] [answered]
Please, please please.

**Comment by s1291** [comment]
I have tried using Webp files (from [link](https://developers.google.com/speed/webp/gallery1?hl=fr)) with both the web app and the CLI, and it appears that Webp is indeed supported.

**Reply by tyt2y3** [reply]
Thanks @s1291 . I don't think the CLI supports webp (at least I don't remember doing anything about it), but yeah browsers can decode webp natively and so may be the web app works out of the box.

**Reply by s1291** [reply, marked as answer]
@tyt2y3 
Here is a screenshot:

![image](https://github.com/visioncortex/vtracer/assets/17838489/a43ecbd8-d9d5-4160-879e-09cdb3fe0997)

I can open the SVG file and edit it in Inkscape.

**Reply by tyt2y3** [reply]
Thanks for letting me know! Yes can confirm the `image` crate actually supported `webp` (at least a common subset of it).

**Reply by PPDEMOone** [reply]
From https://github.com/visioncortex/vtracer/discussions/80

Thank your reply! But when I trying to convert the webp asset, it has working, but the original color of the picture will disappear. 


`
 vtracer --preset photo  --input /Users/ares/Desktop/c18f4b7f-d1ad-4e87-8b7d-8916a0105443-1.webp --output ./test2.svg
`


<img width="887" alt="image" src="https://github.com/visioncortex/vtracer/assets/64009332/5e3a9e11-1b26-40b6-b8a7-c727864acc6a">

**Comment by RunsOnRum** [comment]
Thank you both for being thorough. It appears to not care for the webp file I was using and I apologies for not testing with a control. I have reproduced the results with the file I was working with and receive the following error:

`thread 'main' panicked at 'Conversion failed with error message: No image file found at specified input path' cmdapp\src\main.rs:11:13`

![image](https://github.com/visioncortex/vtracer/assets/9207350/53c57c28-4c69-4d7e-b0e5-807f8deadaed)

I have since tested with the example webp image found on the wiki article on the file format and the conversion was successful.
